google_drive/google_docs.py

```diff
@@ -92,18 +115,24 @@ def studio_submit(self, submissions, suffix=''):
             'result': 'success',
         }
 
+    # suffix argument is specified for xblocks, but we are not using herein
     @XBlock.json_handler
-    def check_url(self, data, suffix=''):
-
+    def check_url(self, data, suffix=''):  # pylint: disable=unused-argument,no-self-use
+        """
+        Checks that the given document url is accessible, and therefore assumed to be valid
+        """
+        test_url = data['url']
```

Comment:
The previous exception catching was too broad, but this one is too narrow -- should catch the cases where there is no url provided, or use a default value.